js/faq.js

```diff
@@ -1,19 +1,23 @@
 const faqTitles = document.querySelectorAll('.faq__title');
+const faqLinks = document.querySelectorAll('.faq_link');
+const currentLocation = window.location.hash;
 
 faqTitles.forEach((faqTitle) => {
   faqTitle.addEventListener('click', (event) => {
-    const currentlyActiveFaqTitle = document.querySelector('.faq__title.show');
+    var currentlyActiveFaqTitle = document.querySelector('.faq__title.show');
```

Comment:
Any reason of going with `var`?

Reply:
It's just a habit.I just forget evertime to use const instead of using var.I know using "const" is a good practice. I will change it and take care of it next time for sure.